### PR TITLE
Warn users about Sentinelsat being broken for official downloads

### PR DESCRIPTION
**Pull request by j08lue**
As discussed in #607

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,6 +1,25 @@
 sentinelsat
 ===========
 
+⚠️ Sentinelsat is currently not functional for downloading from official mirrors ⚠️
```
Comment (valgur) [inline README.rst:4]:
```suggestion
⚠️ Sentinelsat is currently not functional for downloading from the CDSE ⚠️
```

Comment (valgur) [reply README.rst:4]: I would lump the national mirrors under "official mirrors", so that wording seems a bit misleading to me.

File: README.rst [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+
+The `Copernicus Open Access Hub (aka SciHub) <https://scihub.copernicus.eu/>`_, that Sentinelsat was primarily built for, is permanently closed.
+
+.. image:: readme-note-scihub-closed.png
+
+The official successor data distribution system, the `Copernicus Data Space Ecosystem (CDSE) <https://dataspace.copernicus.eu/>`_,
+uses different API logic that Sentinelsat does not (yet?) support. Contributions are welcome to change that, please check issues and discussions
+for ongoing efforts or alternatives.
+
+Sentinelsat can still be used against other data hubs powered by the `DHuS software <https://github.com/SentinelDataHub/>`_.
```
Comment (valgur) [inline README.rst:15]:
```suggestion
Sentinelsat can still be used against [other data hubs](https://github.com/kr-stn/awesome-sentinel#data-hubs-and-national-mirrors) powered by the `DHuS software <https://github.com/SentinelDataHub/>`_.
```

**Review by valgur — CHANGES_REQUESTED** [review]
Two minor changes, other than that LGTM. Thanks!

**Review by avalentino — APPROVED** [review]

**Pull request merged**